docs/data/joy/components/typography/typography.md

```diff
@@ -6,53 +6,71 @@ githubLabel: 'component: Typography'
 
 # Typography
 
-<p class="description">Use typography to present your design and content as clearly and efficiently as possible.</p>
+<p class="description">The Typography component helps present design and content clearly and efficiently.</p>
 
 ## Introduction
 
-The `Typography` component helps you maintain a consistent design by offering a limited set of values to choose while also providing a few convenient props that makes common designs faster to build.
-Joy UI's uses [_Public Sans_](https://fonts.google.com/specimen/Public+Sans?query=public), a Google Font, as the default typeface.
+The Typography component helps maintain a consistent design by providing a limited set of values to choose from and convenient props for building common designs faster.
 
-## Component
-
-After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
+## Basics
 
 ```jsx
 import Typography from '@mui/joy/Typography';
-
-export default function MyApp() {
-  return <Typography>Hello world!</Typography>;
-}
 ```
 
+The Typography component wraps around its content, and displays text with specific typographic styles and properties.
+
+{{"demo": "TypographyBasics.js"}}
+
+### Nested Typography
+
+The Typography component renders as a `<p>` by default.
+Nested Typography components are rendered as `<span>` elements (unless customized by [the `component` prop](#semantic-elements)).
+
+{{"demo": "NestedTypography.js"}}
+
+## Customization
+
 ### System props
 
-As a CSS utility component, `Typography` supports every [`system`](/system/properties/) property.
+As a CSS utility component, Typography supports every [MUI System](/system/properties/) property.
+These properties can be used to customize the styling of the component and make it fit seamlessly with the overall design.
+
+:::warning
+Note that the `color` prop is an exception, it refers to the palette instead of the text color specifically. To set the text color, use the `textColor` prop.
+:::
 
 ```jsx
 <Typography textColor="neutral.500" fontSize="sm" fontWeight="lg">
 ```
 
```

Comment:
Should we be a little more redundant here for the sake of clarity? This is something that I've seen folks asking here and there, and summing the description with these examples feels 100% clear! But really up to ya!

```suggestion

```jsx
<Typography color="neutral" fontSize="sm" fontWeight="lg">
// uses the neutral palette, defaulting to the 500 value

<Typography textColor="neutral.300" fontSize="sm" fontWeight="lg">
// changes this specific element's color to neutral
```
```

Reply:
I like this! But maybe the comments should go above their respective snippets? I don't think we have an explicit best practice here, but that seems to be the convention throughout the docs for the most part.

Reply:
Ah, yeah, for sure ⎯ feel free to format it the best way possible! More concerned about we being super clear about the expected result of each one of these props!

Reply:
@samuelsycamore has this been worked on or do I need to add it?

Reply:
@LadyBluenotes go ahead and incorporate @danilo-leal's suggestion, but with the code comments above rather below the snippets. Then it should be ready to merge!